src/lib/server/RequestHandler.js

```diff
@@ -27,7 +26,7 @@ let _Entry;
  * because the tests move around in folders and it wont exist when this file
  * is initialized but instead when the method is run.
  */
-if (process.env.NODE_ENV !== "test") {
+if (process.env.NODE_ENV === "production") {
```

Comment:
This is potentially controversial since I'm not sure the original reason it was added. As far as I can tell now that `generateStaticHTMLFile` is being imported in build.js this will fail unless NODE_ENV is production anyway.